std/experimental/xml/cursor.d

```diff
@@ -0,0 +1,989 @@
+/*
+*             Copyright Lodovico Giaretta 2016 - .
+*  Distributed under the Boost Software License, Version 1.0.
+*      (See accompanying file LICENSE_1_0.txt or copy at
+*            http://www.boost.org/LICENSE_1_0.txt)
+*/
```

Comment:
As far as I know, we have started to remove those blocks and keeping only what you have below.